webview/src/experiments/components/table/body/commitsAndBranches/CommitsAndBranchesNavigation.tsx

```diff
@@ -17,46 +16,38 @@ export const CommitsAndBranchesNavigation: React.FC = () => {
   )
 
   return (
-    <tbody>
-      <tr>
-        <td className={tableStyles.experimentsTd}>
-          <div className={styles.commitsAndBranchesNav}>
-            {hasMoreCommits && (
-              <button
-                className={styles.commitsAndBranchesNavButton}
-                onClick={showMoreCommits}
-                disabled={isBranchesView}
-              >
-                Show More Commits
-              </button>
-            )}
-            {isShowingMoreCommits && (
-              <button
-                className={styles.commitsAndBranchesNavButton}
-                onClick={showLessCommits}
-                disabled={isBranchesView}
-              >
-                Show Less Commits
-              </button>
-            )}
+    <div className={styles.commitsAndBranchesNav}>
+      {hasMoreCommits && (
+        <button
```

Comment:
Have we already considered/tried making these actions more button-like instead of link-like? 

<img width="878" alt="Screenshot 2023-04-27 at 8 59 25 AM" src="https://user-images.githubusercontent.com/43496356/234885402-20d618e7-c8c9-423e-ae8b-0c25954ca59e.png">

With these actions now more immersed into the table verses at the bottom, I noticed that we don't really use this link design anywhere else in the table 🤔

Reply:
These were buttons first. Maybe @shcheklein can tell more about the reasoning behind the link-like buttons.

Reply:
I don't have any insights on this :) It just felt simpler at that point - full fledged side-panel like button will feel too noisy I think in that amount inside the table. We can definitely / and should think about some icons or lightweight elements / button. Any ideas folks?